testsuite/generate-format/src/api.rs

```diff
@@ -59,6 +59,14 @@ fn trace_crypto_values(tracer: &mut Tracer, samples: &mut Samples) -> Result<()>
     tracer.trace_value(samples, &secp256k1_public_key)?;
     tracer.trace_value(samples, &secp256k1_signature)?;
 
+    let secp256r1_ecdsa_private_key = secp256r1_ecdsa::PrivateKey::generate(&mut rng);
+    let secp256r1_ecdsa_public_key =
+        aptos_crypto::PrivateKey::public_key(&secp256r1_ecdsa_private_key);
+    let secp256r1_ecdsa_signature = secp256r1_ecdsa_private_key.sign(&message).unwrap();
+    tracer.trace_value(samples, &secp256r1_ecdsa_private_key)?;
+    tracer.trace_value(samples, &secp256r1_ecdsa_public_key)?;
+    tracer.trace_value(samples, &secp256r1_ecdsa_signature)?;
+
```

Comment:
I think we should really just make a common and put things there and then call them from here...

Reply:
Same but I think unfortunately `trace_crypto_values` in `consensus.rs` has a check for BLS keypairs / signatures that are not present in `api.rs` and `aptos.rs`. Should they all have the same checks? If so, I can make it so that they all are the same.